src/Bundle/Twig/Node/WebpackNode.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace Hostnet\Bundle\WebpackBundle\Twig\Node;
+
+/**
+ * @author Harold Iedema <hiedema@hostnet.nl>
+ */
+class WebpackNode extends \Twig_Node
+{
+    /** {@inheritdoc} */
+    public function compile(\Twig_Compiler $compiler)
+    {
+        foreach ($this->getAttribute('files') as $file) {
+            $compiler->write('$context["asset"] = "'. $file .'";');
```

Comment:
For the javascript side: The code exported to the webpack config is already escaped.
And the PHP side: The resolver maps the given value - no matter what it is - to an exported file. If that file doesn't exist, it returns false.

If the value is false, the twig node export is skipped (it's not amongst the elements in the $files array). If it is a file, it's a returned value from the resolver, which is always a valid file.